kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -3121,6 +3121,14 @@ object KyuubiConf {
       .timeConf
       .createWithDefaultString("P14D")
 
+  val SERVER_TEMP_FILE_EXPIRE_MAX_COUNT: ConfigEntry[Long] =
+    buildConf("kyuubi.server.tempFile.maxCount")
+      .doc("The upper threshold size of server-side temporary file paths to cleanup")
+      .version("1.10.0")
+      .serverOnly
+      .longConf
+      .createWithDefault(10000)
```

Comment:
how much memory will be occupied?

Reply:
Estimated in a map struct, the total memory consumption would be approximately:
max count 10000 * (key length 13+6 + value length ~50 ) =  2,020,000B ~= 2MB

Reply:
Or do you want me to increase the default value?

Reply:
A potential issue, what will happen if an active file expires.

Reply:
Both time-based and size-based eviction is a fallback approach to clean up the temp files. It's assumed that the threshold is large enough for use.

Reply:
14d might be long enough, 10000 is relatively small, how about disabling the size-based eviction strategy by default?

Reply:
OK. 1. Disabled size-based eviction strategy by default ; 2. Increased the default time-base eviction time to 30days.